infra/helper.py

```diff
@@ -295,9 +295,10 @@ def _env_to_docker_args(env_list):
   return sum([['-e', v] for v in env_list], [])
 
 
-def _workdir_from_dockerfile(dockerfile_path):
-  """Parse WORKDIR from the Dockerfile."""
-  WORKDIR_REGEX = re.compile(r'\s*WORKDIR\s*([^\s]+)')
+def _workdir_from_dockerfile(project_name):
+  """Parse WORKDIR from the Dockerfile for the given project."""
+  WORKDIR_REGEX = re.compile(r'\s*WORKDIR\s*(\$SRC/[^\s]+)')
```

Comment:
why is the addition of $SRC needed in the regex? There are lots of Dockerfiles without $SRC in their WORKDIR command.

Reply:
For example this is wrong:

```
>>> import helper
>>> helper._workdir_from_dockerfile('firefox')
'/src/firefox'
```
```
Should be:
'/src/mozilla-central'
```

Reply:
Yeah :( let me upload another fix.